django_x509/migrations/0007_extensions.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.0 on 2019-12-13 10:04
+
+import collections
+from django.db import migrations
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_x509', '0006_passphrase_field'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='ca',
+            name='extensions',
+            field=jsonfield.fields.JSONField(blank=True, default=list, dump_kwargs={'indent': 4}, help_text='additional x509 certificate extensions', load_kwargs={'object_pairs_hook': collections.OrderedDict}, verbose_name='extensions'),
+        ),
+        migrations.AlterField(
+            model_name='cert',
+            name='extensions',
+            field=jsonfield.fields.JSONField(blank=True, default=list, dump_kwargs={'indent': 4}, help_text='additional x509 certificate extensions', load_kwargs={'object_pairs_hook': collections.OrderedDict}, verbose_name='extensions'),
```

Comment:
For some reason my comment on this one was not sent. can you try editing the existing migration `0001_initial.py` and delete this one?

Since we're not really changing the DB we don't need to add a new migration and would be better to not do so.

You'de need t add the line in which `extensions` is defined.

Reply:
I moved it but I don't understand why this was not there in the migrations already.